src/com/cdd/bao/util/URIRowLine.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * BioAssay Ontology Annotator Tools
+ *
+ * (c) 2014-2016 Collaborative Drug Discovery Inc.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License 2.0
+ * as published by the Free Software Foundation:
+ *
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+package com.cdd.bao.util;
+
+import javafx.scene.Node;
+import javafx.scene.control.Button;
+import javafx.scene.control.TextField;
+import javafx.scene.control.Tooltip;
+
+public class URIRowLine extends RowLine
+{
+	private URITextFieldDelegate delegate;
+
+	private TextField getURITextField()
+	{
+		for (Node n : getChildren())
```

Comment:
The variable name "n" is pretty much exclusively used for integer enumeration, so use something longer, like "node".